discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -97,9 +118,15 @@ def _terraform_params(self, **kwargs):
 
     def _get_specific_tf_entity_params(self) -> Dict[str, str]:
         if isinstance(self._entity_config, BaseClusterConfig):
-            return {"cluster_name": self.entity_name.get(), "cluster_domain": self._entity_config.base_dns_domain}
```

Comment:
Why?

Reply:
This is part of the changes done in @osherdp's PR, but looks okay to me.
the original line is 103 chars long with is not recommended by PEP8: https://realpython.com/python-pep8/#maximum-line-length-and-line-breaking

Reply:
Our linter PEP8 max-line value is 120. You can set it back to how it was

Reply:
Done